### ISSUE TITLE
[OPS] Ability to delete unpublished/unposted submissions

### ISSUE THREAD

**Issue by alexxxmendonca** [pkp/pkp-lib#6982]
**Describe the problem you would like to solve**
As a server administrators, we sometimes make tests. These tests are being "rejected" to they no longer appear in our working queue, giving that there is not a method to delete such test submissions.

However this is not helpful for our statistics as the number of rejected submissions are "inflated" by these tests.

**Describe the solution you'd like**
Ability restricted to server managers for deleting unpublished/unposted submissions. This would help cleaning up test submissions and/or mistakes.

**Who is asking for this feature?**
Server managers.

**Comment by crosfield** [comment]
Support this for OJS too

**Comment by asmecher** [comment]
@crosfield, it should be possible in OJS to send submissions to the archive then delete them from the submission list.

**Comment by NateWr** [comment]
Related to https://github.com/pkp/pkp-lib/issues/4540.

**Comment by alexxxmendonca** [comment]
We have one user who has multiple unsubmitted submissions, they would like to remove them, but they can't:

![image](https://github.com/pkp/pkp-lib/assets/4059679/08931f30-d73b-4a65-8101-4aa0e5a34e23)

Shouldn't this be possible? Aren't these just drafts? They haven't even been submitted yet.

**Comment by asmecher** [comment]
@alexxxmendonca, to clarify, this is an author in OPS (3.3.0-x) who is trying to remove their own incomplete submissions?

**Comment by YvesLepidus** [comment]
I think this issue is related to: [#6271](https://github.com/pkp/pkp-lib/issues/6271)
Which has a [recent update](https://github.com/pkp/pkp-lib/issues/6271#issuecomment-2498453049) that there is NISO work on retractions. 
@asmecher does this functionality of deleting unpublished/unposted submissions depend on this NISO work?

**Comment by asmecher** [comment]
@YvesLepidus, yes, it's related; I've added it to the list on [the main issue where we're tracking this](https://github.com/pkp/pkp-lib/issues/6271#issuecomment-2498453049). When we have a chance to look at this for specification work, we'll look at the related issues holistically.

**Comment by asmecher** [comment]
I think this is resolved as of 3.5.0 with a combination of https://github.com/pkp/pkp-lib/issues/6528 (for incomplete submissions) and the ability to fully delete submissions once rejected. Here's a video of that second process:

https://github.com/user-attachments/assets/cb34bd35-c6b8-457d-b121-5f2e58d04fac

It shows...

1. Rejecting the article; this does result in a decision being recorded and impacts rejection counts. HOWEVER...
2. Deleting the article. This removes all traces of it, including the "reject" decision.

The reason that rejection is necessary first is because the "Delete" action is a serious one and should have some friction. The "Delete" button won't show until the submission has first been rejected.

Closing has not planned -- but this has in fact already been implemented per the video.
